R/zzz.R

```diff
@@ -79,6 +79,13 @@
     vctrs::s3_register("dplyr::dplyr_reconstruct", "resample_results", method = dplyr_reconstruct_resample_results)
     vctrs::s3_register("dplyr::dplyr_reconstruct", "iteration_results", method = dplyr_reconstruct_iteration_results)
   }
+
+  if (newer_tibble()) {
```

Comment:
You may want to add a `# TODO` here that says that when the new version of tibble goes to CRAN, we should:

- bump the required tibble version
- remove these conditional registrations
- register methods with the traditional `@importFrom tibble size_sum` and `@export` system

Reply:
It looks like Kirill is releasing `tibble` in the next few days so we might not need this at all. I'll wait to merge this until that is on CRAN (assuming that it is on CRAN by the 2021-03-01 deadline for `tune`).